### PR TITLE
Crashfix

### PR DESCRIPTION
**Pull request by adrianschroeter**

### CODE REVIEW

File: src/api/app/jobs/issue_tracker_fetch_issues_job.rb [hunk 1]
```diff
@@ -2,6 +2,6 @@ class IssueTrackerFetchIssuesJob < ApplicationJob
   queue_as :issuetracking
 
   def perform(issue_tracker_id)
-    IssueTracker.find(issue_tracker_id).fetch_issues
+    IssueTracker.find_by_id(issue_tracker_id).try(:fetch_issues)
```
Comment (Ana06) [inline src/api/app/jobs/issue_tracker_fetch_issues_job.rb]: `find_by_id` is equivalent to `find`. Please do not use the dynamic version. :see_no_evil:

**Comment by adrianschroeter** [comment]
On Montag, 11. Februar 2019, 15:47:43 CET Ana María Martínez Gómez wrote:
> Ana06 requested changes on this pull request.
> 
> > @@ -2,6 +2,6 @@ class IssueTrackerFetchIssuesJob < ApplicationJob
> 
>    queue_as :issuetracking
> 
>    def perform(issue_tracker_id)
> -    IssueTracker.find(issue_tracker_id).fetch_issues
> +    IssueTracker.find_by_id(issue_tracker_id).try(:fetch_issues)
> 
> `find_by_id` is equivalent to `find`. Please do not use the dynamic version.
> :see_no_evil:

no, it throws an exception when it does not find the tracker

-- 

Adrian Schroeter
SUSE Linux Products GmbH,  Maxfeldstr. 5, 90409 Nuernberg, Germany
email: adrian@suse.de

**Comment by bgeuken** [comment]
I fixed the rubocop offense now. Hope that was okay:-)

**Comment by bgeuken** [comment]
@adrianschroeter Just that you know, there is one failing test now.

**Comment by hennevogel** [comment]
@adrianschroeter do you need help addressing the failed test?

**Comment by Ana06** [comment]
I fixed the tests. @adrianschroeter @bgeuken @hennevogel please check

**Pull request merged**